docs/development.md

```diff
@@ -227,7 +227,30 @@ experimental-features = nix-command flakes
 
 ## Quick Usage Guide
 
-Get a Nix development shell:
+### With `direnv`
+
+The recommended way to develop is to install [`direnv`](https://direnv.net/)
+(usually via `apt` or any other package manager), then `cd` to the repository
+root, and run `direnv allow` in the terminal (only needs to be done the first
```

Comment:
the repository root -> the Oak repository root.

Reply:
Changed to `this repository`